packages/lab/src/__tests__/__e2e__/tabs-next/TabsNext.cy.tsx

```diff
@@ -0,0 +1,293 @@
+            cy.wait(50);
+            cy.realPress("ArrowRight");
+            cy.wait(50);
+            cy.realPress("ArrowRight");
+            cy.findAllByRole("tab").eq(4).should("be.focused");
+            cy.realPress("ArrowRight");
+            cy.findAllByRole("tab").eq(4).should("be.focused");
+          });
+        });
+      });
+    });
+  });
+  describe("WHEN initial size is not sufficient to display all contents", () => {
+    describe("WHEN it initially renders", () => {
+      it("THEN overflow indicator is included in keyboard navigation", () => {
+        cy.mount(<SimpleTabstrip width={320} />);
+        cy.findAllByRole("tab").eq(0).realClick();
+        cy.wait(50);
+        cy.realPress("ArrowRight");
+        cy.wait(50);
+        cy.realPress("ArrowRight");
+        cy.wait(50);
+        cy.realPress("ArrowRight");
+        cy.wait(50);
+        cy.realPress("ArrowRight");
+        cy.findByRole("combobox").should("be.focused");
+      });
+
+      it("THEN overflow indicator opens overflow menu", () => {
+        cy.mount(<SimpleTabstrip width={320} />);
+        cy.findByRole("combobox").focus().realPress("Enter");
+        cy.findByRole("listbox");
+      });
+    });
+  });
+  describe("WHEN overflow is opened", () => {
+    it("THEN overflow menu can be navigated up and down", () => {
+      cy.mount(<SimpleTabstrip width={100} />);
+      cy.findByRole("combobox").click();
+      cy.findByRole("listbox");
+      cy.focused().realPress("ArrowDown");
+      cy.findAllByRole("option")
+        .should("have.length", 4)
+        .eq(1)
+        .should("have.class", "saltHighlighted");
+      cy.focused().then(($container) => {
+        cy.findAllByRole("option")
+          .should("have.length", 4)
+          .eq(1)
+          .then(($el) => {
+            expect($container.attr("aria-activedescendant")).to.equal(
+              $el.attr("id")
+            );
+          });
+      });
+      cy.focused().realPress("ArrowUp");
+      cy.findAllByRole("option")
+        .should("have.length", 4)
+        .eq(0)
+        .should("have.class", "saltHighlighted");
+    });
+    it("THEN overflow menu can be closed with Escape", () => {
+      cy.mount(<SimpleTabstrip width={100} />);
+      cy.findByRole("combobox").click();
+      cy.findByRole("listbox").should("exist");
+      cy.focused().realPress("Escape");
+      cy.findByRole("listbox").should("not.exist");
+    });
+    it("THEN overflow menu can be closed with ArrowLeft and focus is returned to last visible tab", () => {
+      cy.mount(<SimpleTabstrip width={120} />);
+      cy.findByRole("combobox").click();
+      cy.findByRole("listbox").should("exist");
+      cy.focused().realPress("ArrowLeft");
+      cy.findByRole("listbox").should("not.exist");
+      cy.focused().should("have.attr", "role", "tab");
+    });
+    it("THEN overflow menu item can be selected with Enter and focus is returned to the overflow button", () => {
```

Comment:
Is this the intended behaviour? Should active one receive focus?

Reply:
I agree, will change.